wallet-ui-store.html

```diff
@@ -94,7 +94,7 @@ <h6>Wallet Contents:</h6>
       ? vp.verifiableCredential[0]
       : vp.verifiableCredential;
     document.getElementById('credentialType').innerHTML = getCredentialType(vc);
-    document.getElementById('credentialIssuer').innerHTML = vc.issuer;
+    document.getElementById('credentialIssuer').innerHTML = vc.issuer.id;
```

Comment:
Hmm, needs to be more like:

```suggestion
    document.getElementById('credentialIssuer').innerHTML = vc.issuer.id ?? vc.issuer;
```

`issuer` can be a string with the ID value or an object that MUST have an `id` property with the ID as the value.